### ISSUE TITLE
Cop idea: Redundant nil

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#11938]
## Actual behavior

I have the code:

```ruby
module ApplicationHelper
  def my_method
    return nil if 1.positive?
  end
end
```

I run rubocop:

```bash
ydakuka@yauhenid:~/Work/project$ bin/rails_docker rubocop app/helpers/application_helper.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

## Expected behavior

Rubocop suggests to remove redundant `nil` in order to get the following code:

```ruby
module ApplicationHelper
  def my_method
    return if 1.positive?
  end
end
```

## RuboCop version

```bash
ydakuka@yauhenid:~/Work/project$ bin/rails_docker rubocop -V
1.51.0 (using Parser 3.2.1.0, rubocop-ast 1.28.1, running on ruby 2.7.8) [x86_64-linux]
  - rubocop-capybara 2.18.0
  - rubocop-performance 1.17.1
  - rubocop-rails 2.19.1
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.22.0
  - rubocop-thread_safety 0.5.1
```

**Comment by koic** [comment]
This issue will resolve by enabling the existing `Style/ReturnNil` cop:
https://docs.rubocop.org/rubocop/1.52/cops_style.html#examples-201

This cop is disabled by default:
https://github.com/rubocop/rubocop/pull/4638#issuecomment-325219286
